### ISSUE TITLE
Issue : Inconsistent use of custom transforms

### ISSUE THREAD

**Issue by PrinceSajjadHussain** [rasbt/b3-basic-batchsize-benchmark#2]
The `Cifar10DataModule`'s `setup` method uses default transforms if `custom_train_transform` is None, even though `custom_test_transform` might be defined. It should check for the `custom_test_transform` also. This inconsistency can lead to the training set using the default transforms while the test set uses the custom ones (or vice-versa), if only one of the custom transforms is provided.
